File: Examples/Ruby/Attachments/post_add_email_attachment.rb
require 'aspose_email_cloud'

class Attachment

  include AsposeEmailCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("", "")
    @email_api = EmailApi.new
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def post_add_email_attachment
    file_name = "email_test.eml"
    upload_file(file_name)
    attach_name = "README.TXT"
    upload_file(attach_name)

    response = @email_api.post_add_email_attachment(file_name, attach_name)
  end

end

attachment = Attachment.new()
puts attachment.post_add_email_attachment

File: Examples/PHP/Attachments/PostAddEmailAttachment.php
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Email\EmailApi;
use Aspose\Email\AsposeApp;

class Attachment {

	public $email;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->email = new EmailApi();
	}

	public function postAddEmailAttachment() {
		// Upload file to Aspose Cloud Storage
		$name = 'email_test.eml';
		$attachName = "README.TXT";
		Utils::uploadFile($name);
		Utils::uploadFile($attachName);
	
		$result = $this->email->PostAddEmailAttachment($name, $attachName, $storage = "", $folder = "");
		print_r($result);
	}
}

$attachment = new Attachment();
$attachment->postAddEmailAttachment();

?>